api/USERS_GUIDE.md

```diff
@@ -281,4 +281,59 @@ current holds for the project:
     "P1.proj_waiting",
     "P1.proj_avail"
 ]
-```
\ No newline at end of file
+```
+
+### Proofreading
+
+The proofreading APIs require retaining and passing in state information. The
+following examples use [jq](https://jqlang.github.io/jq/) to parse responses
+and store them as environment variables.
+
+Get a project's details and parse out the project state:
+```bash
+curl -s -H "Accept: application/json" -H "X-API-KEY: $API_KEY" \
+    -X PUT "https://www.pgdp.org/api/v1/projects/projectID44de3936807f1" \
+    > project.json
+
+PROJECT_STATE=$(cat project.json | jq -r '.state')
+```
+
+Check out a page and parse out the page name, state, and page text:
+```bash
+curl -s -H "Accept: application/json" -H "X-API-KEY: $API_KEY" \
+    -X PUT "https://www.pgdp.org/api/v1/projects/projectID44de3936807f1/checkout?state=$PROJECT_STATE" \
+    > proof_page.json
+
+PAGE_NAME=$(cat proof_page.json | jq -r .pagename)
+PAGE_STATE=$(cat proof_page.json | jq -r .pagestate)
```

Comment:
I took a look at "Invoking jq" in the manual (wow, that's a long file!), and one of the things it says there is

> Note: it is important to mind the shell´s quoting rules. As a general rule
       it´s best to always quote (with single-quote characters on Unix shells) the
       jq program, as too many characters with special meaning to jq are also
       shell meta-characters.

At line 298 above, you quote `'.state'`, and below, at line 309, you quote `'{text}'`. Is there a reason (or is it optional?) that `.pagename` and `.pagestate` aren't quoted?

To be fair, it's like this in the original example in PR #1208, and it worked (I just didn't notice, and didn't check the manual on jq at the time).

Reply:
The quotes are only required if there are special characters (like `{}` or `$`, etc) so the `'{text}'` requires the quotes and the other places are optional. I'll remove the ones around `'.state'` since those aren't required.

Reply:
This is a total nitpick, so feel free to ignore, but you can always replace things like 
```PAGE_STATE=$(cat proof_page.json | jq -r .pagestate)```
with
```PAGE_STATE=$(jq -r .pagestate < proof_page.json)```
or, if you prefer it
```PAGE_STATE=$(< proof_page.json jq -r .pagestate)```